docs/user/manifests/kubeapps-local-dev-dex-values.yaml

```diff
@@ -3,15 +3,18 @@ certs:
   web:
     create: false
 config:
-  issuer: https://dex.dex:32000
+  # TODO: Don't assume that kind network will be the first non-default bridged network (ie. 172.18.0.2)
+  # This means we'll need to generate the certs as well, rather than pre-configuring them.
+  # Or check if we can use cert generated for tls with k8s node?
+  issuer: https://172.18.0.2:32000
```

Comment:
I am not sure how you got this, I believe that in my case is 172.17.0.2 (I may be wrong and this IP appears when using a NodePort)

Reply:
Do you mean that the `kind` bridge network is `172.17.0.0/16` for you? That's unexpected. Locally, I see the default bridge network `bridge` on that range, while kind's custom network on the next available `172.18.0.0/16`, as shown here:

```
docker network inspect bridge kind --format "{{.Name}} {{.IPAM.Config}}"

bridge [{172.17.0.0/16  172.17.0.1 map[]}]
kind [{172.18.0.0/16  172.18.0.1 map[]} {fc00:f853:ccd:e793::/64  fc00:f853:ccd:e793::1 map[]}]
```

This also matches what's seen in the docker network tutorial for the default `bridge` and the first custom one? https://docs.docker.com/network/network-tutorial-standalone/

Reply:
I don't have a "kind" network (only `bridge`):

```
▶ docker network inspect bridge kind --format "{{.Name}} {{.IPAM.Config}}"
bridge [{172.17.0.0/16  172.17.0.1 map[]}]
Error: No such network: kind
```

Probably something is misconfigured on my side though because between restarts I just re-start the `kind-control-plane` container which is probably not enough. BTW, how do you handle it when you restart your laptop?

Reply:
On both points, I'm guessing the issue is that you're not using the 0.8.x release of kind, where both (breaking) changes were added: https://github.com/kubernetes-sigs/kind/releases :) Sorry, I should have mentioned that was required.